projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.ts

```diff
@@ -21,6 +21,7 @@ let nextUniqueId = 0;
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LgAccordionPanelHeadingComponent implements AfterViewChecked {
+  @Input() iconName = '';
```

Comment:
My inspiration was `LgListWithIconsItemComponent` which uses the same approach, and it felt a bit easier to add a new Input to the accordion instead of adding content projection. But if content projection is a must do, I can update the code.

Reply:
I've actually used the content projection approach after all. The code looks much better than using the @Input iconName.

Reply:
Is iconName here still relevant?

Reply:
I've removed the iconName